webknossos/webknossos/dataset/properties.py

```diff
@@ -368,10 +368,11 @@ def __layer_properties_pre_structure(
         type_value: type[LayerProperties | SegmentationLayerProperties],
     ) -> LayerProperties | SegmentationLayerProperties:
         if d["dataFormat"] == "wkw":
```

Comment:
So this is basically an evolution of all possible input dicts to the current schema?
And this is why we need to check if "wkwResolutions" is present, as it might not be, since the current version does not have it anymore?

Reply:
Yes, that is my understanding also.